### DISCUSSION TITLE
A new name for easysftp

### DISCUSSION THREAD

**Discussion by flamboyantpenguin** [flamboyantpenguin/easysftp#41] [Ideas]
Dear Users

We decided to change our name from easysftp to a name that is _'accessible, friendly and easy to memorize'_. Yes, we are open to suggestions from the public. Your ideas can be posted here. The best three shall be selected and a poll will be conducted to decide the winner. 

Note: While posting the names, do include a description about your name and why it would be a perfect file for _'accessible, friendly and easy to memorize'_. 

Ignite Delta
DAWN/ペンギン

**Comment by jonesrison** [comment]
Swift SFTP

**Reply by flamboyantpenguin** [reply]
Thank you for your valuable suggestion. However it would be better if you could give a brief description about the name and why it would be a perfect fit for _'accessible, friendly and easy to memorize'_. Apologies for not mentioning this before. 

Cheers
DAWN/ペンギン

**Comment by flamboyantpenguin** [comment]
What's your opinion about the name QuickSFTP? 
The word Quick would be something easy to memorize, friendly and perhaps accessible. SFTP signifies our software's main purpose to transfer files using `SFTP`.

**Comment by flamboyantpenguin** [comment]
The poll has been started at #43. New ideas are still welcome till the end of the poll.
